test/unit/foreman/renderer/snapshots/ProvisioningTemplate/user_data/Preseed_Autoinstall_cloud-init_user_data.ubuntu_autoinst4dhcp.snap.txt

```diff
@@ -31,7 +31,9 @@ autoinstall:
   storage:
     layout:
       name: lvm
+
   late-commands:
+
```

Comment:
these empty lines look odd, especially the the second one.

Reply:
Yeah, what should I say: building pretty-formated YAML out of templates with conditional includes/snippets is meh.

If the snippet is inserted, then we definitely need a newline to produce valid YAML/cloud-config. But if not we get the newline anyway because of `<%= indent(2) { snippet ... if ... } %>`. is no option here.

Alternatively, we could add a blank line at the end of the snippet and use  `<%= ... -%>`. Tell me what's the best way to solve this.

Reply:
Yeah. Templating YAML is awful, I agree.
I think I do prefer the `-%>` way, but wouldn't block on this if you find it too ugly :)